### ISSUE TITLE
 C___U_s_e_r_et_al folder name in postprocessing

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#1783]
Since quite some time now, I see that the automatically created folder name in the COCO postprocessing is ` C___U_s_e_r_et_al` on my Windows machine:
![image](https://user-images.githubusercontent.com/13706996/40044816-d63ea396-5828-11e8-8634-30724b129096.png)

A while ago, this folder name was showing the algorithms (actually algorithm name abbreviations) that were used in the comparison. That was much more helpful...

**Comment by islandblue** [comment]
This part seems relevant.

https://github.com/numbbo/coco/blob/c83c49b628253a0d708e1bb1f8f625853963ffd3/code-postprocessing/cocopp/findfiles.py#L136-L144

@brockho can you share what command/code you run to get this?

**Comment by nikohansen** [comment]
The offending line is probably a few lines above

```python
if isinstance(args, str):
```

**Comment by nikohansen** [comment]
May be fixed with 344ad20, @brockho could you test this?

**Comment by brockho** [comment]
For the record, it was a simple algorithm, I postprocessed to get the error. Something like
```
import cocopp
cocopp.main('NEWUOA!')
```

With the latest fix in the development branch, the error is gone. Thanks @nikohansen!
